ibmdbpy/tests/test_statistics.py

```diff
@@ -41,6 +41,18 @@ def test_idadf_numeric_stats_default(self, idadf):
         assert isinstance(_numeric_stats(idadf, "min", columns), numpy.ndarray)
         assert isinstance(_numeric_stats(idadf, "max", columns), numpy.ndarray)
 
+    def test_idadf_numeric_stats_one_column(self, idadf):
+        data = idadf._table_def() # We necessarly have to put the test under this condition
+        allcolumns = list(data.loc[data['VALTYPE'] == "NUMERIC"].index)
+        columns = [allcolumns[0]]
```

Comment:
Rather than going through extracting the first element of `allcolumns` just to put it into a list again, you can [slice](https://docs.python.org/2/tutorial/introduction.html#lists) the list you're building into a list containing only a single element:

```python
columns = data.loc[data['VALTYPE'] == "NUMERIC"].index.tolist()[:1]
```